src/model/Series.ts

```diff
@@ -545,22 +545,19 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
             seriesName = '';
         }
         seriesName = seriesName
-            ? encodeHTML(seriesName) + (!multipleSeries ? newLine : ': ')
+            ? encodeHTML(seriesName) + (!multipleSeries ? newLine : ' ')
             : '';
 
         colorStr = typeof colorEl === 'string' ? colorEl : colorEl.content;
         const html = !multipleSeries
-            ? seriesName + colorStr
+            ? seriesName + (seriesName ? '<br/>' : '') + '<p style="margin: 8px 0 0;">' + colorStr
```

Comment:
in my opinion, do not use `<p>`, in case that p is defined with other css properties like `line-height`, padding, and so on.
It's OK to use div.